osu.Game/Screens/IOsuScreen.cs

```diff
@@ -60,8 +60,9 @@ public interface IOsuScreen : IScreen
 
         /// <summary>
         /// Whether mod track adjustments are allowed to be applied.
+        /// A <see langword="null"/> value means that the parent screen's value of this setting will be used.
         /// </summary>
-        bool AllowTrackAdjustments { get; }
+        bool? AllowTrackAdjustments { get; set; }
```

Comment:
Can't say I'm overly happy with this public setter myself. Is setting it to internal, or having the fallback logic occur in `OsuScreen` itself (by resolving the music controller) not viable?

Reply:
This is actually a great idea, can move all the "remember from previous screen" logic into `OsuScreen` and not use nullable in the interface this way. Pushed the change